incremental_ext_test.go

```diff
@@ -0,0 +1,38 @@
+package feedx
+
+import (
+	"errors"
+	"time"
+
+	"github.com/bsm/bfs"
+)
+
+func (p *IncrementalProducer) LoadManifest() (*Manifest, error) {
```

Comment:
wait, what are you doing? you should just `return p.loadManifest()` not duplicate everything again